src/network/clientpackethandler.cpp

```diff
@@ -1474,3 +1475,15 @@ void Client::handleCommand_ModChannelSignal(NetworkPacket *pkt)
 	if (valid_signal)
 		m_script->on_modchannel_signal(channel, signal);
 }
+
+void Client::handleCommand_CameraModeRestrictionFlags(NetworkPacket *pkt)
+{
+	u16 byteflags;
+	*pkt >> byteflags;
+
+	CameraModeRestrictionFlags *flags = new CameraModeRestrictionFlags;
```

Comment:
parenthesis please

Reply:
Done. Also added a ctor for `CameraModeRestrictions`